.buildkite/dagster-buildkite/dagster_buildkite/steps/packages.py

```diff
@@ -735,17 +735,27 @@ def tox_factors_for_folder(tests_folder_name: str) -> List[str]:
             f"{deps_factor}-{command_factor}"
             for deps_factor in ["dbt17", "dbt18", "dbt19", "dbt110"]
             for command_factor in ["cloud", "core-main", "core-derived-metadata"]
-        ],
+        ]
+        + ["dbtfusion-snowflake"],
```

Comment:
tangent: should we trim some older versions as we add newer ones?

Reply:
our policy is to test against all dbt-core versions that are not marked as EOL: https://docs.getdbt.com/docs/dbt-versions/core

looks like we've been a bit lax on bumping this up, and we could potentially remove dbt18 now. That should actually simplify a good chunk of code that's currently gating on that version. However, it doesn't seem like a terrible policy to extend support a bit longer than EOL for now